executor/executor_xapi.go

```diff
@@ -591,10 +596,10 @@ func columnNameToPBExpr(client kv.Client, column *ast.ColumnNameExpr, tn *ast.Ta
 	}
 	// If the column ID is in fields, it means the column is from an outer table,
 	// its value is available to use.
-	return datumToPBExpr(client, *column.Refer.Expr.GetDatum())
+	return b.datumToPBExpr(client, *column.Refer.Expr.GetDatum())
 }
 
-func datumToPBExpr(client kv.Client, d types.Datum) *tipb.Expr {
+func (b *executorBuilder) datumToPBExpr(client kv.Client, d types.Datum) *tipb.Expr {
```

Comment:
But at least it is consistent with other ToPBExpr functions.
Also, it may be used in the future.